spec/service_consumers/pact_helper.rb

```diff
@@ -38,7 +41,7 @@ def url_encode(str)
   provider_state "a service exists called number-plate-supplier with places" do
     set_up do
       service = create(:service, slug: "number-plate-supplier")
-      create(:place, service_slug: service.slug, latitude: 1.11, longitude: 2.22)
+      create(:place, service_slug: service.slug, latitude: 50.742754933617285, longitude: -1.9552618901330387)
```

Comment:
This change probably shouldn't be in a7e583ef798b3702f0e5446ed0b312757f057960 - it should be fixed up to the commit where the pact helper was added.

Reply:
Moved this as well as the Database cleaner addition higher up the code into 34b5ccb 👍